docs/api_guide/tutorial_pynapple_nwb.py

```diff
@@ -0,0 +1,177 @@
+#
+# 
+# !!! note
+# 	Creating the NWB file is outside the scope of pynapple. The NWB file used here has already been created before.
+# 	Multiple tools exists to create NWB file automatically. You can check [neuroconv](https://neuroconv.readthedocs.io/en/main/), [NWBGuide](https://nwb-guide.readthedocs.io/en/latest/) or even [NWBmatic](https://github.com/pynapple-org/nwbmatic).
+
+
+data = nap.load_file("../../your/path/to/MyProject/sub-A2929/A2929-200711/pynapplenwb/A2929-200711.nwb")
+
+print(data)
+
+# %%
+# Pynapple will give you a table with all the entries of the NWB file that are compatible with a pynapple object.
+# When parsing the NWB file, nothing is loaded. The `NWBFile` keeps track of the position of the data whithin the NWB file with a key. You can see it with the attributes `key_to_id`.
+
+data.key_to_id
+
+
+# %%
+# Loading an entry will get pynapple to read the data.
+
+z = data['z']
+
+print(data['z'])
+
+# %%
+# Internally, the `NWBClass` has replaced the pointer to the data with the actual data.
+#
+# While it looks like pynapple has loaded the data, in fact it did not. By default, calling the NWB object will return an HDF5 dataset.
+
+print(type(z.values))
+
+# %%
+# Notice that the time array is always loaded.
+
+print(type(z.index.values))
+
+# %%
+# This is very useful in the case of large dataset that do not fit in memory. You can then get a chunk of the data that will actually be loaded.
+
+z_chunk = z.get(670, 680) # getting 10s of data.
+
+print(z_chunk)
+
+# %%
+# Data are now loaded.
+
+print(type(z_chunk.values))
+
+# %%
+# You can still apply any high level function of pynapple. For example here, we compute some tuning curves without preloading the dataset.
+
+tc = nap.compute_1d_tuning_curves(data['units'], data['y'], 10)
+
+print(tc)
+
+
+# %%
+# To change this behavior, you can pass `lazy_loading=False` when instantiating the `NWBClass`.
+path = "../../your/path/to/MyProject/sub-A2929/A2929-200711/pynapplenwb/A2929-200711.nwb"
+data = nap.NWBFile(path, lazy_loading=False)
+
+z = data['z']
+
+print(type(z.d))
+
+
+# %%
+# Numpy memory map
+# ----------------
+#
+# In fact, pynapple can work with any type of memory map. Here we read a binary file with [`np.memmap`](https://numpy.org/doc/stable/reference/generated/numpy.memmap.html).
+
+eeg_path = "../../your/path/to/MyProject/sub-A2929/A2929-200711/A2929-200711.eeg"
+frequency = 1250 # Hz
+n_channels = 16
+f = open(eeg_path, 'rb') 
+startoffile = f.seek(0, 0)
+endoffile = f.seek(0, 2)
+f.close()
+bytes_size = 2
+n_samples = int((endoffile-startoffile)/n_channels/bytes_size)
+duration = n_samples/frequency
+interval = 1/frequency
+
+fp = np.memmap(eeg_path, np.int16, 'r', shape = (n_samples, n_channels))
+timestep = np.arange(0, n_samples)/frequency
```

Comment:
maybe you should say what is this example, like this eeg file what format is it? if it is something standard you can point to a link in which they explain how to mem-map the data, and explain something like:
"here we are creating a mem map of an eeg recording from xxxx system, following the recipe at url-to-website"